install/index.md

```diff
@@ -17,8 +17,24 @@ redirect_from:
 - /engine/installation/
 - /en/latest/installation/
 - /linux/
+toc_max: 2
 ---
 
+Docker Engine is an open source containerization technology for building and
+containerizing your applications. Docker Engine acts as a client-server
+application with:
+* A server with a long-running daemon process ([`dockerd`](https://docs.docker.com/engine/reference/commandline/dockerd/)).
+* A REST API which specifies interfaces that programs can use to talk to and
+  instruct the Docker daemon.
+* A command line interface (CLI) client ([`docker`](https://docs.docker.com/engine/reference/commandline/cli/)).
+
+The CLI uses the Docker REST API to control or interact with the Docker daemon
+through scripting or direct CLI commands. Many other Docker applications use the
+underlying API and CLI. The daemon creates and manage Docker objects, such as
+images, containers, networks, and volumes.
+
+## Releases
+
 Docker Engine - Community is ideal for developers and small
```

Comment:
Wondering if we should list the "edge" and "stable" channels for docker desktop separately 🤔

Reply:
Removed the reference to Community. 

Desktop Edge and Stable release info is available in the Desktop topics. As there are links to Desktop in the Supported platforms section, thought it might be better to not repeat this info here. Please let me know if you think otherwise.